### ISSUE TITLE
amdgpu not offered on R9 390 and catalyst/ati break

### ISSUE THREAD

**Issue by Achati** [manjaro/mhwd#26]
i installed manjaro on my desktop after having no problems with oit on my laptop
but mhwd only offers catalyst, ati and vesa drivers and autoconfiguring any of those breaks my system in some way or another on xorg start

also amdgpu is not even listed or available (even though the mhwd-amdgpu package is installed)
and i installed and configured it manually

maybe it is because my graphics card is a R9 390 but it is detected as a R9 290

i suspect for what i want to do using amdgpu-pro wouild be the best but that is not supported and the package conflicts with mhwd (which would break the system)

PS: if i have time i can try to replicate the issues with ati and catalyst drivers after using the autoconfiguration feature

**Comment by philmmanjaro** [comment]
I need more specifics and some hardware information to help you out. What is the output of:

```
mhwd
mhwd -lh
hwinfo --gfx
```

**Comment by Achati** [comment]
i will have tme tomorrow to turn my desktop on (haven't been using it for a while since manjaro on my laptop works better and is portable)
update it and replicate the issue and get the output
i will then edit this comment

**Comment by NikkyAI** [comment]
i bought a rx 480 and will put my old one on the shelf for now, lets hope that solves the problem for me

**Comment by mirh** [comment]
@NikkyAI did it?

**Comment by NikkyAI** [comment]
yeah it fixed it.. since then i upgraded to vega 64 (and a new load of problems)
